common/addressing/defaults.go

```diff
@@ -49,13 +50,15 @@ var (
 	// StateIPv6Mask represents the CIDR Mask for the state position.
 	StateIPv6Mask = net.CIDRMask(112, 128)
 
-	// IPv6 prefix length for address assigned to container. The default is
-	// L3 only and thus /128.
+	// ContainerIPv6Mask IPv6 prefix length for address assigned to
+	// container. The default is L3 only and thus /128.
 	ContainerIPv6Mask = net.CIDRMask(128, 128)
-	// IPv4 prefix length for address assigned to container. The default is
-	// L3 only and thus /32
+	// ContainerIPv4Mask IPv4 prefix length for address assigned to
```

Comment:
There are a couple of other instances as well. Can you make sure these are all complete sentences?